script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -109,14 +109,14 @@ The mere thought of returning to such an anachronistic lifestyle makes me feel d
 The train arrives at a large station and stops for several minutes.
 Since this train is stopping at every station, I had to wait for the next rapid train to pass by.
 <Page30>
-There is no one else on the platform.
+No one's there on the platform.//I think the line is meant to say that there's actually no one on the platform.
```

Comment:
Meaning change is good, but I'd just phrase it as `There is no one on the platform.`. 

> ホームに人影はない。